### PR TITLE
Project community kickoff

### PR DESCRIPTION
**Pull request by puerco**
Signed-off-by: Adolfo García Veytia (Puerco) <adolfo.garcia@uservers.net>

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+according to the maintainer consensus process.
+
+## Maintainer Consensus
+
+To reach a decision on an issue, proposal, or formal OVEP, the proponents need
+to seek maintainer consensus. In the context of this document, "maintainer
+consensus" means collecting a sufficient number of favorable opinions of
+community members with maintainer status to move forward with a proposal.
+
+This document goes not prescribe a method of voting. Any mechanism that enables
```
Comment (luhring) [inline GOVERNANCE.md:50]:
```suggestion
This document does not prescribe a method of voting. Any mechanism that enables
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+- Any community member may self-nominate as a maintainer candidate after
+actively contributing to OpenVEX constantly for six months or more.
+- A sponsoring committee of maintainers that meets the qualifying criteria may
+nominate any community member as a candidate.
+
+### Sponsoring Committees
+
+To nominate a community member as a maintainer candidate, a group of
+maintainers may file a nomination. The commute shall meet 5he following criteria
```
Comment (luhring) [inline GOVERNANCE.md:109]:
```suggestion
maintainers may file a nomination. The commute shall meet the following criteria
```

Also, what does "the commute" mean here?

File: GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+to be qualified to file the nomination:
+
+- The number of members in the committee shall not be less than two (2).
+- Whenever the number of organizations with maintainers in the project is more
+than three(3), committee members shall be from different affiliations.
```
Comment (luhring) [inline GOVERNANCE.md:114]:
```suggestion
than three (3), committee members shall be from different affiliations.
```

Also, does "affiliations" mean "organizations"? (Wondering if it'd be more clear to say "organizations" in that case)

**Review by luhring — APPROVED** [review]
Nice! See nit comments but looks great.

**Comment by puerco** [comment]
@luhring PTAL i've repushed it with the nits fixed.

**Pull request merged**